### DISCUSSION TITLE
How to open twitter hyperlinks to tweets in squawker

### DISCUSSION THREAD

**Discussion by dln949** [j-fbriere/squawker#66] [Q&A]
On my android device, when reading an article in the web broswer that contains a hyperlink to a twitter tweet, when I tap on the hyperlink, it opens Squawker as expected; however, Squawker appears to assume the hyperlink is to a twitter user's profile page instead of to the specified tweet, so Squawker squawks an error message at me saying the user cannot be found.  Well, of course the user cannot be found, because the hyperlink is to a tweet.  

What do I have to do in Squawker - and/or in my android settings - so that tapping on a hyperlink to a tweet will open that tweet in Squawker?

**Comment by j-fbriere** [comment]
What is the link that causes the error? I will have a look at it.

**Comment by dln949** [comment]
https://t.co/drRRQ49gQ9

**Comment by j-fbriere** [comment]
This is a problem with shorten twitter links ( https://t.co/... ) that must be fixed in Squawker.
It will be fixed in the next release of Squawker.

**Reply by dln949** [reply]
But when I "unshorten" it to be https://twitter.com/EAlexander46/status/1698862295500128391/photo/1 , and then tap on that complete hyperlink, Squawker still doesn't find the correct twitter post.  Squawker simply opens to the default "home" page.

**Reply by j-fbriere** [reply]
Yes it's because the actual link ends with /photo/1, which Squawker did not manage well.
If you remove this last part of the URL it will work.
I made a fix about that along with the shorten link fix.

**Comment by j-fbriere** [comment]
The version [3.4.4](https://github.com/j-fbriere/squawker/releases/tag/v3.4.4) fix the launch of Squawker from shorten links and also links that have additional parts (eg. /photo/1).
